### PR TITLE
Remove `getProfile` calls when loading feed

### PR DESCRIPTION
**Pull request by haileyok**
## Why

Currently, we make a ton of calls to `getProfile()` to display the "Reply to" label. However, since https://github.com/bluesky-social/atproto/pull/2461, we can now get this info from the `grandparentAuthor` field in the API response.

### Before

Screenshot 2024-05-06 at 4 20 52 PM

### After

Screenshot 2024-05-06 at 4 21 12 PM

Verify that the requests are no longer present when loading your timeline. Also verify that posts properly display the `Reply to` label when they should.

Check situations where:

- It appears normally: i.e. a post with one parent above. It should show the parent's parent's author.
- We remove the parent from the timeline. It should show the parent's author.
- Where we put a thread together, i.e. below

### A normal situation

Screenshot 2024-05-06 at 5 19 31 PM

### Removing the parent

Screenshot 2024-05-06 at 5 18 51 PM

### Us making a thread 

Screenshot 2024-05-06 at 5 16 31 PM

### CODE REVIEW

File: src/state/queries/preferences/const.ts
Comment (haileyok) [inline src/state/queries/preferences/const.ts:37]: Not sure if this is the correct move here

File: package.json
Comment (estrattonbailey) [inline package.json:54]: We should not merge this update unless we also plan to get movable following feed in in the same release. Once we merge this, anyone on the TF will also start diverging from v1->v2 feeds prefs.

If we need to get this in though, there are options, just need to discuss.

**Review by estrattonbailey — CHANGES_REQUESTED** [review]
Just blocking for now on the SDK update, looks great otherwise

File: package.json
Comment (haileyok) [reply package.json:54]: Perfect, we were wondering what that was and were going to hold off anyway so good to know. Thank you!

**Review by gaearon — APPROVED** [review]

**Pull request merged**
